isovar/variant_reads.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2016. Mount Sinai School of Medicine
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Functions for getting AlleleReads which support variant alleles.
+"""
+
+from .variant_helpers import trim_variant
+from .allele_reads import reads_overlapping_variant, reads_overlapping_variants
+
+def filter_non_alt_reads_for_variant(variant, allele_reads):
+    _, _, alt = trim_variant(variant)
+    return [read for read in allele_reads if read.allele == alt]
+
+def filter_non_alt_reads_for_variants(variants_and_allele_reads_sequence):
```

Comment:
Huh, I guess it used to be used in `protein_sequences` but now I'm doing it via groupby_allele instead. I think it's still worth keeping in case someone only cares about total vs. alt reads.